scripts/export_auth_token.bash

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+## Usage: eval $(./export_auth_token.bash -c [CREDENTIAL_HASH] -a [ACCOUNT_NAME])
+
+## 1. First, export your credentials hash:
+##    export CREDENTIALS=`echo -n "username:password" | md5sum | cut -d ' ' -f 1`
+##    OR
+##    Supply the hash as an argument to the script
+## 2. eval the script as in the USAGE note above
+## 3. Profit!
+
+command -v jq >/dev/null 2>&1 || { echo >&2 "This script requires 'jq' be installed"; exit 1; }
+
+usage() { echo 'Usage: eval $('"$0"' [-c {CREDENTIALS_HASH}] [-a {ACCOUNT_NAME}] [-p {PHONE_NUMBER}] [-r {ACCOUNT_REALM}])' 1>&2;}
+
+function authenticate() {
+    local C="$1"
+    local TYPE="$2"
+    local ID="$3"
+    AUTH_RESP=$(curl -s -X PUT http://localhost:8000/v2/user_auth -d "{\"data\":{\"credentials\":\"$C\", \"$TYPE\":\"$ID\"}}")
+
+    STATUS=$(echo $AUTH_RESP | jq -r '.status')
```

Comment:
jq isn't listed because this script isn't necessarily part of normal kazoo installation/operation. I've included it because it was handy for me, when testing, to regenerate the auth token and still be able to re-run curl commands by using the env variables instead of the literal values. That's the whole point of the script, to set auth_token and account_id into env variables.